variants/logos_arachnio_due/variant.cpp

```diff
@@ -25,39 +25,72 @@ const Port Ports[NUM_PORTS]=
- * |            | LEDs & button    |        |                 |
- * +------------+------------------+--------+-----------------+--------------------------------------------------------------------------------------------------------
- * | 0          | N/A              |  PC10  | LED1            |
- * | 1          | N/A              |  PC17  | LED2            |
- * | 2          | N/A              |  PA5   | BP2             | WKUP4
- * +------------+------------------+--------+-----------------+--------------------------------------------------------------------------------------------------------
-*/
-  { PORTC, PIO_PC10, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER },
-  { PORTC, PIO_PC17, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER },
-  { PORTA, PIO_PA5, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER }, // WKUP4
-
-/* +------------+------------------+--------+-----------------+--------------------------------------------------------------------------------------------------------
- * |            | Serial           |        |                 |
- * +------------+------------------+--------+-----------------+--------------------------------------------------------------------------------------------------------
- * | 7          |                  |  PA9   | ICE USB Serial  | URXD0
- * | 7          |                  |  PA10  | ICE USB Serial  | UTXD0
- * +------------+------------------+--------+-----------------+--------------------------------------------------------------------------------------------------------
-*/
-  { PORTA, PIO_PA9A_URXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER }, // URXD0
-  { PORTA, PIO_PA10A_UTXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER }, // UTXD0
-
+	{ PORTA, PIO_PA9A_URXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D0/URXD0
+	{ PORTA, PIO_PA10A_UTXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 	// D1/UTXD0
+	{ PORTA, PIO_PA3A_TWD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D2/SDA
+	{ PORTA, PIO_PA4A_TWCK0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D3/SCL
+	{ PORTB, PIO_PB3A_UTXD1, GPIO_PERIPH_A, ADC_Channel7, NOT_ON_PWM, NOT_ON_TIMER },			// D4
+	{ PORTB, PIO_PB4, GPIO_NOMUX, ADC_Channel7, NOT_ON_PWM, NOT_ON_TIMER },						// D5
+	{ PORTB, PIO_PB2A_URXD1, GPIO_PERIPH_A, ADC_Channel6, NOT_ON_PWM, NOT_ON_TIMER },			// D6
+	{ PORTB, PIO_PB4, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D7
+	{ PORTB, PIO_PB7, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D8
+	{ PORTB, PIO_PB5, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D9
+	{ PORTB, PIO_PB2, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D10
+	{ PORTB, PIO_PB1, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D11
+	{ PORTB, PIO_PB11, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D12
```

Comment:
Good catch... predictably, it should be PA11.
On Sep 3, 2015 8:06 AM, "Thibaut VIARD" <notifications@github.com> wrote:

> In variants/logos_arachnio_due/variant.cpp
> <https://github.com/aethaniel/ExperimentalCore-sam/pull/3#discussion_r38656131>
> :
>
> > -  { PORTA, PIO_PA9A_URXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER }, // URXD0
> > -  { PORTA, PIO_PA10A_UTXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM,  NOT_ON_TIMER }, // UTXD0
> > -
> > +	{ PORTA, PIO_PA9A_URXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D0/URXD0
> > +	{ PORTA, PIO_PA10A_UTXD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 	// D1/UTXD0
> > +	{ PORTA, PIO_PA3A_TWD0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D2/SDA
> > +	{ PORTA, PIO_PA4A_TWCK0, GPIO_PERIPH_A, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER }, 		// D3/SCL
> > +	{ PORTB, PIO_PB3A_UTXD1, GPIO_PERIPH_A, ADC_Channel7, NOT_ON_PWM, NOT_ON_TIMER },			// D4
> > +	{ PORTB, PIO_PB4, GPIO_NOMUX, ADC_Channel7, NOT_ON_PWM, NOT_ON_TIMER },						// D5
> > +	{ PORTB, PIO_PB2A_URXD1, GPIO_PERIPH_A, ADC_Channel6, NOT_ON_PWM, NOT_ON_TIMER },			// D6
> > +	{ PORTB, PIO_PB4, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D7
> > +	{ PORTB, PIO_PB7, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D8
> > +	{ PORTB, PIO_PB5, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D9
> > +	{ PORTB, PIO_PB2, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D10
> > +	{ PORTB, PIO_PB1, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D11
> > +	{ PORTB, PIO_PB11, GPIO_NOMUX, No_Analog_Channel, NOT_ON_PWM, NOT_ON_TIMER },				// D12
>
> Isn't PB11 USB/DP?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aethaniel/ExperimentalCore-sam/pull/3/files#r38656131>
> .
>